coalib/parsing/DefaultArgParser.py

```diff
@@ -39,7 +39,7 @@ def default_arg_parser(formatter_class=None):
 To find out what kind of analysis coala offers for the languages you use, visit
 http://coala.io/languages, or run::
 
-    $ coala --show-bears --filter-by-language C Python
+    $ coala --show-bears --filter-by C Python
```

Comment:
Replace `--filter-by-language` -> `--filter-by language` and not just `--filter-by`